docs/Ingestor/PythonSDK.md

```diff
@@ -1,9 +1,84 @@
 # Use of Python SDK
 
-You find an example script in the demo folder https://github.com/paulscherrerinstitute/scilog/tree/main/demo . The example will add a paragraph text and a second paragraph including two figures.
+You find an example script in the demo folder https://github.com/paulscherrerinstitute/scilog/tree/main/demo/demo.py . The example will add a paragraph containg tet, images and tags as well querying the existing snippet data in a logbook.
```

Comment:
there's a typo here `tet` instead of `test`

Reply:
Thanks for fixing